docs/pages/includes/yaml-lint-note.mdx

```diff
@@ -0,0 +1,6 @@
+<Admonition type="note" title="YAML formatting">
```

Comment:
@ptgott should we get rid of this admonition? It looks more like something that should be in a guide than in a reference.

Reply:
IMHO it could only help or not help, but it doesn't hurt anything being where it is in `teleport-kube-agent.mdx`.

Reply:
As far as I know, Helm returns YAML parsing errors to the user in `helm install`, so it acts as the linter that this Admonition is recommending that you use. I only think we need this if there's a values file setting that is particularly tricky to write as YAML.